.github/workflows/qiita-ci.yml

```diff
@@ -169,17 +169,20 @@ jobs:
           conda deactivate
 
           echo "8. Setting up SSH"
-          ssh-keygen -t rsa -b 4096 -N '' -f $PWD/qiita_ware/test/test_data/test_key
+          ssh-keygen -t ed25519 -a 200 -N '' -f $PWD/qiita_ware/test/test_data/test_key
           mkdir ~/.ssh/
           cp $PWD/qiita_ware/test/test_data/test_key* ~/.ssh/
           cat ~/.ssh/test_key.pub > ~/.ssh/authorized_keys
+          cat ~/.ssh/test_key.pub > ~/.ssh/authorized_keys2
           chmod 600 $PWD/qiita_ware/test/test_data/test_key*
```

Comment:
Where does `authorized_keys2` get used?

Reply:
I think this should help: https://serverfault.com/a/116193. Similarly to the previous change, it's just to be sure that we have all the possible things to make this work.